### PR TITLE
Fixed bug#1190343

### PR DESCRIPTION
**Pull request by lvicoun**
### PR creator: Description

Describe the overall goals of this pull request.

### PR creator: Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [ ] SLE 15 SP4/openSUSE Leap 15.4 *(current `main`, no backport necessary)*
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [ ] SLE 15 GA
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4
  - [ ] SLE 12 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/storage_filesystems.xml [hunk 1]
```diff
@@ -600,13 +613,15 @@ Id Path      single   single    single   Unallocated
        step. A subvolume can either be identified by path (for example
        <filename>/var/tmp</filename>) or by <literal>0/<replaceable>SUBVOLUME
        ID</replaceable></literal> (for example <literal>0/272</literal>). The
-       following example sets a quota of 5 GB for <filename>/var/tmp</filename>.
+       following example sets a quota of 5 GB for
```
Comment (lproven) [inline xml/storage_filesystems.xml:616]:
```suggestion
       following example sets a quota of 5&nbsp;GB for
```

File: xml/storage_filesystems.xml [hunk 2]
```diff
@@ -623,78 +638,79 @@ Id Path      single   single    single   Unallocated
...
   <sect2 xml:id="sec-filesystems-major-btrfs-swapping">
-	  <title>Swapping on Btrfs</title>
-    <important>
-     <title>Snapshots with swapping</title>
+   <title>Swapping on Btrfs</title>
+   <important>
+    <title>Snapshots with swapping</title>
+    <para>
+     You will not be able to create a snapshot if there is any enabled swap
+     file in the source subvolume.
```
Comment (lproven) [inline xml/storage_filesystems.xml:668]:
```suggestion
     You will not be able to create a snapshot if the source subvolume contains
     any enabled swap files.
```

Comment (lproven) [reply xml/storage_filesystems.xml:668]: The phrasing sounded a little odd to me, but I am open to further suggestions!

**Review by lproven — CHANGES_REQUESTED** [review]
Very minor stuff. Sorry for the delay.

File: xml/storage_filesystems.xml (hunk 2 above)
Comment (lvicoun) [reply xml/storage_filesystems.xml:668]: I like your suggestion.

**Pull request closed without merging**